sjb/config/test_cases/test_pull_request_origin_extended_conformance_k8s.yml

```diff
@@ -1,15 +1,13 @@
 ---
-parent: 'common/test_cases/origin_release.yml'
+parent: 'test_cases/test_branch_origin_extended_conformance_gce.yml'
 overrides:
-  junit_analysis: False
```

Comment:
why? being pushed to gubernator, right?

Reply:
it's set in the base

Reply:
Or do I also have to override in a descendant?

Reply:
oh no sorry didn't see that it was in ancestor